motioneye/sendtelegram.py

```diff
@@ -111,7 +102,7 @@ def on_media_files(media_files):
     if not timespan:
         return on_media_files([])
 
-    logging.debug('waiting for pictures to be taken')
+    logging.debug('waiting %ds for pictures to be taken' % float(timespan))
```

Comment:
Preferring f-strings over %-formatting:
```suggestion
    logging.debug(f'waiting {float(timespan)}s for pictures to be taken')
```